.gitignore

```diff
@@ -23,7 +23,7 @@
 hs_err_pid*
 replay_pid*
 
-scripts/git-config.sh
+git-config.sh
```

Comment:
Shouldn't this line be removed entirely if you removed the `git-config.sh` script?